fhir-persistence-jdbc/README-JDBC-Performance-Tests.md

```diff
@@ -0,0 +1,43 @@
+# IBM FHIR Server - fhir-persistence-jdbc performance tests
+
+Performance tests for derby embedded, derby network and db2.
```

Comment:
Very minor, but I like the oxford comma :-P
```suggestion
Performance tests for derby embedded, derby network, and db2.
```

Reply:
done